universe/node/api.go

```diff
@@ -19,6 +20,12 @@ import (
 func (n *Node) RegisterAPI(r *gin.Engine) {
 	n.log.Infof("Registering api for node: %s...", n.GetID())
 
+	if n.cfg.Common.DisableCORS {
```

Comment:
Can you flip the boolean around? Just very confusing like this :)

So how about `AllowCORS`, since you are enabling/allowing Cross Origin Requests.
Default it to False, with the (dev) config example setting it to true? (so nothing changes for deploy envs)

Reply:
Yes, `allow` is better
Changed